website/addons/onedrive/tests/test_models.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not going to go through every test yet, but please update this file, as well as `test_serializers` and `test_views` to use the mixins that provide automated tests found in `website/addons/base/testing/`. See aforementioned test files for Box, Dropbox, or GoogleDrive for closest approximations of expected testing behaviors.